specification/informatica/Informatica.DataManagement/main.tsp

```diff
@@ -303,7 +303,7 @@ model CompanyDetails {
 @doc("Marketplace details.")
 model MarketplaceDetails {
   @doc("Marketplace Subscription Id")
-  marketplaceSubscriptionId: string;
+  marketplaceSubscriptionId?: string;
```

Comment:
Is your intension here to make this field optional for both PUT and Patch request? I wonder what is your service behavior, if this field is null in request for PUT request for new resource creation.

Reply:
We marked this field as required by mistake. Ideally the Marketplace subscription Id is created after the Marketplace create call..